### ISSUE TITLE
Accessibility issue in Terra-Slide-Panel

### ISSUE THREAD

**Issue by SushmithaBN** [cerner/terra-framework#850]
# Bug Report

## Description
In the latest version of terra-slide-panel, we can see few accessibility violations when it is inspected [here](https://engineering.cerner.com/terra-framework/raw/tests/terra-slide-panel/slide-panel/slide-panel-fullscreen)
The same violations are seen while running WDIO test for accessibility

## Additional Context / Screenshots
![image](https://user-images.githubusercontent.com/42537313/63282778-31bc0280-c2cd-11e9-9cd0-e7a9ffd0c19f.png)

**Comment by ShettyAkarsh** [comment]
cc @neilpfeiffer

**Comment by scottwilmarth** [comment]
@ShettyAkarsh or @SushmithaBN what elements are in question for this JIRA? I see the AXE screenshot with the "Panel Content" heading highlighted, but that passes required contrast ratios. Even when I open the originally tested panel link "here" originally listed above, all of the content passes color contrast ratios. Please provide more context to the issue.

**Comment by lokesh-0813** [comment]
@scottwilmarth The `axe` violation occurs because the panel content is set to `isFullScreen`. And since it is full screen, panel content is overlapping main content and the main content is not visible for axe tool.
